src/components/Vendor.vue

```diff
@@ -19,13 +19,13 @@ import { vendors } from "../lib/vendors";
         </thead>
 
         <tbody>
-          <tr v-for="(vendor, i) in vendors.data" :key="i">
-            <template
-              v-if="
-                vendor.fields.Status === 'Active' &&
-                vendor.fields.Section === section
-              "
-            >
+          <tr
+            v-for="vendor in vendors.data.filter(
+              (v) => v.fields.Section === section
+            )"
+            :key="vendor.id"
+          >
+            <template v-if="vendor.fields.Status === 'Active'">
```

Comment:
@JordanPisani I missed this... it should also go in the filter statement above

`v.fields.Section === section && v.fields.Status === 'Active'`

Reply:
Should I approve this PR, pull the changes locally, make the change, then push them out? Or is there an easier way?